### ISSUE TITLE
Tens of errors when used with other listings code

### ISSUE THREAD

**Issue by asdrsdg** [wg030/jlcode#34]
Hi, I am using this amazing package but I wanted to also have some JSON code in my paper. I got inspired by [this ](https://tex.stackexchange.com/questions/83085/how-to-improve-listings-display-of-json-files) and wanted to use some of the code but whenever I use their code together with jlcode I get up to  60 errors like 
```
Package xcolor: Undefined color jlrule
Package xcolor: Undefined color jlop
Package xcolor: Undefined color jlbase ...
```
MWE:
```
\documentclass{article}
\usepackage{bera}% optional: just to have a nice mono-spaced font
\usepackage{listings}
\usepackage{xcolor}

\colorlet{punct}{red!60!black}
\definecolor{background}{HTML}{EEEEEE}
\definecolor{delim}{RGB}{20,105,176}
\colorlet{numb}{magenta!60!black}

\lstdefinelanguage{json}{
    basicstyle=\normalfont\ttfamily,
    numbers=left,
    numberstyle=\scriptsize,
    stepnumber=1,
    numbersep=8pt,
    showstringspaces=false,
    breaklines=true,
    frame=lines,
    backgroundcolor=\color{background},
    literate=
     *{0}{{{\color{numb}0}}}{1}
      {1}{{{\color{numb}1}}}{1}
      {2}{{{\color{numb}2}}}{1}
      {3}{{{\color{numb}3}}}{1}
      {4}{{{\color{numb}4}}}{1}
      {5}{{{\color{numb}5}}}{1}
      {6}{{{\color{numb}6}}}{1}
      {7}{{{\color{numb}7}}}{1}
      {8}{{{\color{numb}8}}}{1}
      {9}{{{\color{numb}9}}}{1}
      {:}{{{\color{punct}{:}}}}{1}
      {,}{{{\color{punct}{,}}}}{1}
      {\{}{{{\color{delim}{\{}}}}{1}
      {\}}{{{\color{delim}{\}}}}}{1}
      {[}{{{\color{delim}{[}}}}{1}
      {]}{{{\color{delim}{]}}}}{1},
}

\begin{document}

\begin{lstlisting}[language=json,firstnumber=1]
{"menu": {
  "id": "file",
  "value": "File",
  "popup": {
    "menuitem": [
      {"value": "New", "onclick": "CreateNewDoc()"},
      {"value": "Open", "onclick": "OpenDoc()"},
      {"value": "Close", "onclick": "CloseDoc()"}
    ]
  }
}}
0123456789
\end{lstlisting}

\begin{jllisting}
thisbreaks
\end{jllisting}
\end{document}
```

I also tried moving the code, removing the \usepackage{xcolor} etc. but nothing helps. Do you know how could I fix this?

**Comment by wg030** [comment]
🤣
The reason your code does not work is that you forgot to include the package.


Add ` \usepackage[autoload=false]{jlcode}` to your preamble and use it like
```
\begin{jllisting}[language=julia, style=jlcodestyle]
thisbreaks
5.0 + "test"
\end{jllisting}
```
and you will be fine.

**Comment by asdrsdg** [comment]
> The reason your code does not work is that you forgot to include the package.

Oh that is just my mistake when I copy-pasted the code in here from my IDE... 

> Add \usepackage[autoload=false]{jlcode} to your preamble and use it like

You were right... The solution is the autoload option. I'll be honest, when I saw this repository for the first time, I skipped reading the warning about autoload in README. I somehow thought it is only for advanced users. I copypasted the simplest looking code (the one without autoload and other parameters) and it just worked so I thought that 'autoload' is not relevant for me and I forgot about it. Thanks for your time and package, it works great now!
